src/main.js

```diff
@@ -150,19 +159,11 @@ function getRandomIndex(array) {
  }
 
 
- function randomPosterButton() {
-   imageInput.innerText = getRandomIndex(images);
+ function randomPoster() {
```

Comment:
function names should generally start with an action verb, like `generateRandomPoster`